api/fixtures/activity1.yml

```diff
@@ -21,8 +21,8 @@ App\Entity\ScheduleEntry:
 # Root node
 App\Entity\ContentNode\ColumnLayout:
   columnLayout1:
-    root: '@self'
     parent: null
+    root: '@self'
```

Comment:
I think you don't need to change the order of the properties.
(i reverted this could still run the CreateColumnLayout tests).

I would have preferred it if we did not have this unnecessary change in the history.

Reply:
True - thx